packages/headless/src/controllers/core/tab-manager/headless-core-tab-manager.ts

```diff
@@ -0,0 +1,60 @@
+import {configuration} from '../../../app/common-reducers';
+import {CoreEngine} from '../../../app/engine';
+import {tabSetReducer as tabSet} from '../../../features/tab-set/tab-set-slice';
+import {ConfigurationSection, TabSection} from '../../../state/state-sections';
+import {loadReducerError} from '../../../utils/errors';
+import {
+  buildController,
+  Controller,
+} from '../../controller/headless-controller';
+
+export interface TabManager extends Controller {
+  /**
+   * The state of the `TabManager` controller.
+   */
+  state: TabManagerState;
+}
+
+/**
+ * A scoped and simplified part of the headless state that is relevant to the `TabManager` controller.
+ */
+export interface TabManagerState {
+  activeTab: string;
+}
+
+/**
+ * Creates a `TabManager` controller instance.
+ *
+ * @param engine - The headless engine.
+ * @returns A `TabManager` controller instance.
+ */
+export function buildCoreTabManager(engine: CoreEngine): TabManager {
```

Comment:
Doesn't really need to be documented, and if it is it should be marked as @internal